cdap-app-fabric/src/main/java/io/cdap/cdap/internal/tethering/PeerInfo.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright © 2021 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.internal.tethering;
+
+import java.util.Objects;
+import javax.annotation.Nullable;
+
+/**
+ * Peer information that's persisted in the tether store.
+ */
+public class PeerInfo {
+  private final String name;
+  private final String endpoint;
+  private final TetheringStatus tetheringStatus;
+  private final PeerMetadata metadata;
+  private final long lastConnectionTime;
+
+  public PeerInfo(String name, @Nullable String endpoint, TetheringStatus tetheringStatus, PeerMetadata metadata) {
+    this(name, endpoint, tetheringStatus, metadata, 0);
+  }
+
+  public PeerInfo(String name, @Nullable String endpoint, TetheringStatus tetheringStatus,
+                  PeerMetadata metadata, long lastConnectionTime) {
+    this.name = name;
+    this.endpoint = endpoint;
+    this.tetheringStatus = tetheringStatus;
+    this.metadata = metadata;
+    this.lastConnectionTime = lastConnectionTime;
+  }
+
+  public String getName() {
+    return name;
+  }
+
+  @Nullable
+  public String getEndpoint() {
+    return endpoint;
+  }
+
+  public TetheringStatus getTetherStatus() {
```

Comment:
Getter method should follow the naming convention, getTetheringStatus

Reply:
fixed